packages/xo-web/src/xo-app/xosan/new-xosan.js

```diff
@@ -62,6 +63,50 @@ const _findLatestTemplate = templates => {
 const DEFAULT_BRICKSIZE = 100 * 1024 * 1024 * 1024 // 100 GiB
 const DEFAULT_MEMORY = 2 * 1024 * 1024 * 1024 // 2 GiB
 
+const XOSAN_SR_COLUMNS = [
+  {
+    itemRenderer: sr => (
+      <Link to={`/srs/${sr.id}/general`}>{sr.name_label}</Link>
+    ),
+    name: _('xosanName'),
+    sortCriteria: 'name_label',
+  },
+  {
+    itemRenderer: (sr, { hosts }) => {
+      const host = find(hosts, ['id', sr.$container])
+      return <Link to={`/hosts/${host.id}/general`}>{host.name_label}</Link>
```

Comment:
Using `render-xo-item` should do just fine, I don't think it's crucial to be able to sort by host within a pool.